types/models.d.ts

```diff
@@ -222,7 +222,7 @@ declare module 'mongoose' {
     /** Creates a `countDocuments` query: counts the number of documents that match `filter`. */
     countDocuments(
       filter?: FilterQuery<TRawDocType>,
-      options?: QueryOptions<TRawDocType>
+      options?: (mongodb.CountOptions & MongooseSpecificQueryOptions) | null
```

Comment:
if i understand this function correctly, then `lean` (and likely `timestamps`) are not options for this right? (maybe also other options)